packages/api/tests/serial/endpoints/test-granules.js

```diff
@@ -61,6 +63,43 @@ async function runTestUsingBuckets(buckets, testFunction) {
   }
 }
 
+/**
+ * helper for cleaning up after move files tests.
+ * @param {string} publicBucket - public bucket created in setupBucketsConfig
+ */
+async function teardownBuckets(publicBucket) {
```

Comment:
Should this be called in an `afterEach` hook?

Reply:
And correspondingly `setupBucketsConfig` in a `beforeEach` - or is the goal to be conservative on calling these operations?

Reply:
I think it would be in a beforeEach If I pulled out those two tests that use it into a separate file. 
there may be too many different things being tested in this file.  It ended up like this because when I added a second test that moved files, I discovered the old test was leaving files around, so I extracted the duplicate code and added a cleanup.

It could go into the beforeEach/afterEach here, but seems like it would be unused for the majority of the tests.